src/service/AppealStorageService.ts

```diff
@@ -0,0 +1,33 @@
+import axios, { AxiosRequestConfig } from 'axios';
+
+import { Appeal } from 'app/models/Appeal';
+import { getEnvOrThrow } from 'app/utils/EnvironmentUtils';
+
+export class AppealStorageService {
+
+    public async save(appealData: Appeal, token: string): Promise<any> {
+
+        if (token == null) {
+            throw new Error('Token is missing');
+        }
+
+        const config: AxiosRequestConfig = {
+            headers: {
+                Accept: 'application/json',
+                Authorization: 'Bearer ' + token
+            }
+        };
+
+        const apiUri = getEnvOrThrow(`APPEALS_API_URL`);
```

Comment:
Application should fail on start when config is missing not at runtime. Would it help to move it outside of the class so that config is read as soon as application started?